aiaccel/hpo/algorithms/nelder_mead_algorithm.py

```diff
@@ -55,10 +58,17 @@ def __init__(
 
     def get_vertex(self) -> npt.NDArray[np.float64]:
         with self.lock:
-            vertex = next(self.generator)
+            while True:
+                vertex = next(self.generator)
+
+                if vertex is None:
+                    raise NelderMeadEmptyError(
+                        "Cannot generate new vertex now. Maybe get_vertex is called in parallel."
+                    )
 
-        if vertex is None:
-            raise NelderMeadEmptyError("Cannot generate new vertex now. Maybe get_vertex is called in parallel.")
+                if all(normalize_range[0] < x < normalize_range[1] for x in vertex):
```

Comment:
現状でグローバル変数による固定であれば、ハードコードで良いように思います。
```suggestion
                if all(0 <= x <= 1 for x in vertex):
```

Reply:
対応しました

7023efb0e417ee3ee4f9ba60a5b663d6b6351ec6